prepackPy/splitter.py

```diff
@@ -1,33 +1,94 @@
 #!/usr/bin/env python
 
 import numpy as np
-import warnings
 import pandas as pd
 
+
 def splitter(X, target_index, split_size, seed): 
     """
     split a dataset into train and test sets
 
     Parameters
     ----------
-    X: numpy array, pandas dataframe
-        an input dataset
+    X: numpy ndarray, pandas dataframe
+        input dataset
     target_index: integer
-        a column index of target variable in X
+        column index of target variable in X
     split_size: float
-        a proportion of the dataset to include in the test split
+        proportion of the dataset to include in the test split
     seed: integer
-        a random state that will make code reproducible
+        random state that will make code reproducible
 
     Returns
     -------
-    X_train: numpy array
+    X_train: numpy ndarray
         splitted features for model training
-    X_test: numpy array 
-        splitted features for model testing
-    y_train: 1d numpy array
+    y_train: numpy ndarray
         splitted target for model training
-    y_test: 1d numpy array
+    X_test: numpy ndarray 
+        splitted features for model testing
+    y_test: numpy ndarray
         splitted target for model testing
+        
+    Example
+    -------
+    >>> import numpy as np
+    >>> import pandas as pd
+    >>> from splitter import splitter # to do
```

Comment:
I'm a bit confused about the `to do` comment here!

Reply:
Thanks for pointing that. Aha, my bad. I will fix it.